tests/e2e/classic_machine_pool_test.go

```diff
@@ -606,6 +606,54 @@ var _ = Describe("Create MachinePool", ci.Day2, ci.NonHCPCluster, ci.FeatureMach
 		Expect(mpResponseBody.InstanceType()).To(Equal("m5.2xlarge"))
 	})
 
+	It("can create machinepool with customized tags - [id:73942]", ci.High, ci.NonHCPCluster, func() {
+		By("Create machinepool creation plan with invalid tags")
+		invalidTags := map[string]string{
+			"aws:tags": "awsvalue",
+		}
+		replicas := 0
+		machineType := "r5.xlarge"
+		name := "mp-73942"
+
+		MachinePoolArgs := &exe.MachinePoolArgs{
+			Cluster:     &clusterID,
+			Replicas:    &replicas,
+			MachineType: &machineType,
+			Name:        &name,
+			Tags:        &invalidTags,
+		}
+		output, err := mpService.Apply(MachinePoolArgs, false)
+		Expect(err).To(HaveOccurred())
+		Expect(output).Should(ContainSubstring("Tags that begin with 'aws:' are reserved"))
+
+		By("Create a machinepool with variable aws_tags")
+		validTags := map[string]string{
+			"tagsKey": "tagValue",
+		}
+		MachinePoolArgs = &exe.MachinePoolArgs{
+			Cluster:     &clusterID,
+			Replicas:    &replicas,
+			MachineType: &machineType,
+			Name:        &name,
+			Tags:        &validTags,
+		}
+		_, err = mpService.Apply(MachinePoolArgs, false)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Check the machinepool detail state")
+		resp, err := cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, name)
+		Expect(err).ToNot(HaveOccurred())
+		for tagKey, tagValue := range validTags {
+			Expect(tagValue).To(BeElementOf(resp.AWS().Tags()[tagKey]))
+		}
+
+		By("Update the machinepool tags is not allowed")
```

Comment:
This should be in another case or ?

Reply:
@radtriste It's a too small checkpoint. Let's have it in the case and split in future if it is large. WDYT?